File: ServiceBook/ServiceBook/Scripts/Service/Models/ServiceIntervention.js
function ServiceIntervention(data)
{
    this.id=null;
    this.name=null;
    this.price=null;
    this.flag = null;
    this.currency = null;
    this.wpid = null;
    if(data!=null)
    {
        this.currency = new Currency(data.Currency);
        this.id = data.ID;
        this.name = data.Name;
        this.price=data.Price;
        this.flag = data.Flag;
        this.wpid = data.WPID;
    }
}

File: ServiceBook/ClassLibrary1/Models/ServiceIntervention.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServiceBook.DAL.Models
{
   public class ServiceIntervention
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string Name { get; set; }

        public double Price { get; set; }
        
        public int Flag { get; set; }

        public int CID { get; set; }

        [ForeignKey("CID")]
        public virtual Currency Currency { get; set; }

        public int? WP { get; set; }
        [ForeignKey("WP")]
        public virtual WorkingPoint WorkingPoint { get; set; }

        public virtual ICollection<SSI> SSI { get; set; }
    }
}
